File: app/src/main/java/com/karandeep/travelguide/MainActivity.java
package com.karandeep.travelguide;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    ImageView jaipur;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        jaipur=findViewById(R.id.jaipur);
        jaipur.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i=new Intent(getApplicationContext(),HeritageSites.class);
                startActivity(i);
            }
        });
    }
}

File: HelloWorldApp/app/src/main/java/demoapp/codingblocks/com/helloworldapp/MainActivity.kt
package demoapp.codingblocks.com.helloworldapp

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun getIntent(): Intent {
        return super.getIntent()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
